libs/chlorophyll/scss/components/list/_mixins.scss

```
@@ -73,7 +65,7 @@ $border-color: tokens.$border-color;
       flex: 1 1 auto;
     }
 
-    dd {
+    dl {
```

It should be like this here
```suggestion
    dd {
```